### ISSUE TITLE
Terminal translation throws error on Waterfall

### ISSUE THREAD

**Issue by diogotcorreia** [diogotcorreia/Triton#91]
**Describe the bug**
When using Waterfall, the plugin doesn't load because of a bug related not finding a class needed for terminal translation.

**To Reproduce**
Use the latest waterfall and try to load Triton.

**Server information (please complete the following information):**
 - Server Minecraft Version: any
 - Client Minecraft Version (if using ViaVersion or something similar): any
 - BungeeCord (specify if it's a fork): yes, waterfall
 - Triton Version: 2.6.0

**Additional context**
```java
[20:16:11] [main/WARN]: Error enabling plugin Triton
java.lang.NoClassDefFoundError: net.md_5.bungee.log.ConciseFormatter
    at java.lang.J9VMInternals.prepareClassImpl(Native Method) ~[?:1.8.0_202]
    at java.lang.J9VMInternals.prepare(J9VMInternals.java:302) ~[?:1.8.0_202]
    at java.lang.Class.getDeclaredConstructor(Class.java:719) ~[?:1.8.0_202]
    at net.md_5.bungee.api.plugin.PluginManager.enablePlugin(PluginManager.java:363) ~[Bungee.jar:git:Waterfall-Bootstrap:1.15-SNAPSHOT:4339bb7:317]
    at net.md_5.bungee.api.plugin.PluginManager.loadPlugins(PluginManager.java:270) ~[Bungee.jar:git:Waterfall-Bootstrap:1.15-SNAPSHOT:4339bb7:317]
    at net.md_5.bungee.BungeeCord.start(BungeeCord.java:270) ~[Bungee.jar:git:Waterfall-Bootstrap:1.15-SNAPSHOT:4339bb7:317]
    at net.md_5.bungee.BungeeCordLauncher.main(BungeeCordLauncher.java:62) ~[Bungee.jar:git:Waterfall-Bootstrap:1.15-SNAPSHOT:4339bb7:317]
    at net.md_5.bungee.Bootstrap.main(Bootstrap.java:15) ~[Bungee.jar:git:Waterfall-Bootstrap:1.15-SNAPSHOT:4339bb7:317]
Caused by: java.lang.ClassNotFoundException: net.md_5.bungee.log.ConciseFormatter
    at net.md_5.bungee.api.plugin.PluginClassloader.loadClass0(PluginClassloader.java:53) ~[Bungee.jar:git:Waterfall-Bootstrap:1.15-SNAPSHOT:4339bb7:317]
    at net.md_5.bungee.api.plugin.PluginClassloader.loadClass(PluginClassloader.java:27) ~[Bungee.jar:git:Waterfall-Bootstrap:1.15-SNAPSHOT:4339bb7:317]
    at java.lang.ClassLoader.loadClass(ClassLoader.java:862) ~[?:1.8.0_202]
    ... 8 more
```

**Comment by diogotcorreia** [comment]
Triton v2.6.1 will hide the error and allow Triton to load correctly but will disable terminal translation for Waterfall. This might be fixed in a future version.

**Comment by diogotcorreia** [comment]
Terminal translation now works on Waterfall. Will be available in v2.6.2.
